src/sagas/reown.js

```diff
@@ -473,11 +473,27 @@ export function* processRequest(action) {
       case SendNanoContractTxError: {
         yield put(setNewNanoContractStatusFailure());
 
-        const retry = yield call(
-          retryHandler,
-          types.REOWN_NEW_NANOCONTRACT_RETRY,
-          types.REOWN_NEW_NANOCONTRACT_RETRY_DISMISS,
-        );
+        const dontRetryErrors = [
+          'Invalid blueprint ID',
+          'Error getting blueprint id with',
+        ];
+
+        let shouldIgnoreRetry = false;
+        for (let i = 0; i < dontRetryErrors.length; i += 1) {
+          if (e.message.indexOf(dontRetryErrors[i]) > -1) {
+            shouldIgnoreRetry = true;
+            break;
+          }
+        }
+
+        let retry = false;
+        if (!shouldIgnoreRetry) {
```

Comment:
suggestion(non-blocking): This took me a while to understand, as `retry` and `ignore` are already words that carry negative meanings, and the `!` made this more confusing.

I suggest renaming `shouldIgnoreRetry` to `shouldRetry`.

```js
let shouldRetry = true;
... if (e.message.indexOf() > -1) {
  shouldRetry = false;
  break;
} ...

...

let retry = false;
if (shouldRetry) {
  retry = yield call(...)
}
```

Maybe it would be even better to rename `retry` to `retryHandlerResponse` to better inform what this variable carries.